forc-pkg/src/manifest.rs

```diff
@@ -110,7 +110,13 @@ impl Manifest {
     /// Produces the string of the entry point file.
     pub fn entry_string(&self, manifest_dir: &Path) -> anyhow::Result<Arc<str>> {
         let entry_path = self.entry_path(manifest_dir);
-        let entry_string = std::fs::read_to_string(&entry_path)?;
+        let entry_string = std::fs::read_to_string(&entry_path).map_err(|e| {
+            anyhow!(
+                "failed to validate path from entry point {:?}: {}",
```

Comment:
Is there a better way of wording this?

Reply:
Do we not want to specify that it's attempting to validate a path exists from the entry field?

Reply:
Ah good point. In that case make it more wordy, to describe that it's checking the entry point field in the manifest file?